validator/xsd/schedule.xml.xsd

```diff
@@ -242,4 +242,17 @@
     </xs:simpleContent>
   </xs:complexType>
 
+  <xs:simpleType name="optoutEnum">
+    <xs:restriction base="xs:string">
+      <!-- Recording optout status 'false' is the normal case: We enable the live stream and create a recording, which we publish without and feedback from the speaker -->
```

Comment:
Readability/understanding: `which we publish without and feedback from the speaker` could be `which we publish without or with feedback from the speaker`